lib/identifiers/isbn.rb

```diff
@@ -39,22 +39,23 @@ class ISBN
       \d{1,7}   # ISBN title enumerator and check digit
       \b
     }x
-    TEXT_AFTER_PREFIX_REGEXP = ':?\s*(\d.*)$'.freeze
+    TEXT_AFTER_PREFIX_REGEXP = /:?\s*(\d[^a-zA-Z]*)/
```

Comment:
Could you please add comments to each element of the regexp, as done in the regular expressions above?

Reply:
`X` is a valid character in an ISBN, check the regular expressions above. With this regular expression, an ISBN like `ISBN:2-7594-0269-X` will not be extracted

Reply:
If this regexp is going to be used with `Regexp::IGNORECASE`, I see no point in having both lower and upper case in it.